lua/lscs/autorun/cl_forcepowers.lua

```diff
@@ -404,3 +406,6 @@ hook.Add( "PlayerBindPress", "!!!!_lscs_playerbindpress", function( ply, bind, p
 		return true
 	end
 end )
+
+hook.Add( "LSCS:PlayerCanForce", "lscs_canforce", function( ply, power )
```

Comment:
doesnt this hook exist in lscs_forcepowers.lua already?

Reply:
Outdated, fixed in latest.